mlflow/tracking/fluent.py

```diff
@@ -1089,6 +1089,7 @@ def log_params(
 def log_input(
     dataset: Optional[Dataset] = None,
     context: Optional[str] = None,
+    *,
     tags: Optional[Dict[str, str]] = None,
```

Comment:
@harupy Didn't `log_input()` exist before, so this would be a breaking change?

Reply:
It did. It has `dataset` and `context` parameters. `tags` and `model` are new arguments. This won't be a breaking change.

Reply:
I see, `model` is a new argument.